tests/main/test_homePage.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+
+@pytest.mark.django_db
+class TestHomepageView:
+    def test_user_redirected_to_admin_login(self, client):
+        response = client.get('/admin')
+        assert response.status_code == 301
+
+    def test_extends_of_base(self, client):
+        response = client.get('/')
+        assert response.status_code == 200
+        assert b'<nav' in response.content
```

Comment:
Maybe we should check the template name instead? I think it should be in `response.template_name`

Reply:
Done